content/en/docs/setup/best-practices/multiple-zones.md

```diff
@@ -4,401 +4,138 @@ reviewers:
-
-```yaml
-kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Pod
-metadata:
-  name: mypod
-spec:
-  containers:
-    - name: myfrontend
-      image: nginx
-      volumeMounts:
-      - mountPath: "/var/www/html"
-        name: mypd
-  volumes:
-    - name: mypd
-      persistentVolumeClaim:
-        claimName: claim1
-EOF
-```
-
-Note that the pod was automatically created in the same zone as the volume, as
-cross-zone attachments are not generally permitted by cloud providers:
-
-```shell
-kubectl describe pod mypod | grep Node
-```
-
-```shell
-Node:        kubernetes-minion-9vlv/10.240.0.5
-```
-
-And check node labels:
-
-```shell
-kubectl get node kubernetes-minion-9vlv --show-labels
-```
-
-```shell
-NAME                     STATUS    AGE    VERSION          LABELS
-kubernetes-minion-9vlv   Ready     22m    v1.6.0+fff5156   beta.kubernetes.io/instance-type=n1-standard-2,failure-domain.beta.kubernetes.io/region=us-central1,failure-domain.beta.kubernetes.io/zone=us-central1-a,kubernetes.io/hostname=kubernetes-minion-9vlv
-```
-
-### Pods are spread across zones
-
-Pods in a replication controller or service are automatically spread
-across zones.  First, let's launch more nodes in a third zone:
-
-GCE:
-
-```shell
-KUBE_USE_EXISTING_MASTER=true MULTIZONE=true KUBERNETES_PROVIDER=gce KUBE_GCE_ZONE=us-central1-f NUM_NODES=3 kubernetes/cluster/kube-up.sh
-```
-
-AWS:
-
-```shell
-KUBE_USE_EXISTING_MASTER=true MULTIZONE=true KUBERNETES_PROVIDER=aws KUBE_AWS_ZONE=us-west-2c NUM_NODES=3 KUBE_SUBNET_CIDR=172.20.2.0/24 MASTER_INTERNAL_IP=172.20.0.9 kubernetes/cluster/kube-up.sh
-```
-
-Verify that you now have nodes in 3 zones:
-
-```shell
-kubectl get nodes --show-labels
-```
-
-Create the guestbook-go example, which includes an RC of size 3, running a simple web app:
+## Node behavior
 
-```shell
-find kubernetes/examples/guestbook-go/ -name '*.json' | xargs -I {} kubectl apply -f {}
-```
+Kubernetes automatically spreads the Pods for a
+workload resources (such as {{< glossary_tooltip text="Deployment" term_id="deployment" >}}
+or {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}})
+across different nodes in a cluster. This spreading helps
+reduce the impact of failures.
 
-The pods should be spread across all 3 zones:
+When nodes start up, the kubelet on each node automatically adds
+{{< glossary_tooltip text="labels" term_id="label" >}} to the the Node object
+that represents that specific kubelet in the Kubernetes API.
+These labels can include
+[zone information](/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesiozone)
 
-```shell
-kubectl describe pod -l app=guestbook | grep Node
-```
+If your cluster spans multiple zones or regions, you can use node labels
+in conjunction with
+[Pod topology spread constraints](/docs/concepts/workloads/pods/pod-topology-spread-constraints/)
+to control how Pods are spread across your cluster among fault domains.
+such as regions, zones, and even specific nodes.
```

Comment:
Check the sentences (rebase mixup).
`how Pods are spread across your cluster among fault domains.
such as regions, zones, and even specific nodes.`